src/GithubFilesBundle/Controller/DefaultController.php

```diff
@@ -19,18 +19,22 @@ class DefaultController extends Controller
     /**
      * @Route("/user/github/repos")
      */
-    public function reposAction()
+    public function getReposAction(Request $request)
     {
+        $page = $request->query->get('page');
+        $perPage = $request->query->get('perPage');
+
         $currentUser = $this->getUser();
         $response = new JsonResponse();
         $token = new \Milo\Github\OAuth\Token($currentUser->getGithubToken());
```

Comment:
We should probably check that the user has a token (at least isn't empty) before trying to use it.

If the user does not have a token then we can give a better message to the user than "an error occurred or..." and just say that they need to log in.

Also, please document the procedure on registering a client with GitHub and what the correct value of the github_redirect_uri parameter should be.